css/style.css

```diff
@@ -0,0 +1,136 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Poppins;
+  src: url(../fonts/Poppins/Poppins-Regular.ttf);
+}
+
+@font-face {
+  font-family: Poppins-Medium;
+  src: url(../fonts/Poppins/Poppins-Medium.ttf);
+}
+
+body {
+  background-color: #f7f7f9;
+}
+
+header {
+  padding: 0.25rem 1.5rem 0.25rem 0.75rem;
+}
+
+header h4.logo {
+  font-family: Poppins, sans-serif;
+  font-size: 1.125rem;
+  font-style: normal;
+  line-height: 1.25rem;
+  color: #6070ff;
+}
+
+.logo-menubar {
+  display: flex;
+  align-items: flex-start;
+  gap: 12.438rem;
+  padding: 0.25rem 1.5rem 0.25rem 0.75rem;
+}
+
+.presentation-part {
+  background-image: url(../img/header-shapes-mobile/header-shapes\ mobile.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 114px 24px;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 0.75rem;
+}
+
+.presentation-part h1 {
+  display: flex;
+  width: 20.438rem;
+  height: 9.375rem;
```

Comment:
- [ ] According to the project requirements it's best practice to Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels or rem kindly fix this.

Reply:
Dear @ademibowale 
Thank you so much for this information, I have also been in doubt about 
using rem or percentages. Now, I consider the importance of percentage for 
a responsive design. I updated the stylesheet. 
Best regards,